### DISCUSSION TITLE
The Calibre deck tarball

### DISCUSSION THREAD

**Discussion by gilgamsh** [ucb-bar/hammer#862] [Support (Q&A)] [answered]
When I try to download the Calibre deck tarball from http://asap.asu.edu/asap/ according to https://github.com/The-OpenROAD-Project/asap7 . I find that they remove the download link and I have to make request to download them.  I have already sent the request several months, but with no reply.

I am wondering, what should be in the Calibre deck tarball . Can I make it by myself?

**Comment by harrisonliew** [comment]
The Calibre decks contain all the SVRF commands needed to extract the layout for LVS and compare them against the design rules for DRC. These commands are proprietary, so there is no way for you to reproduce them. Many others have also had no luck getting the deck tarball, and we are not allowed to share them either.

**Reply by gilgamsh** [reply]
Is there any way that i can bypass the check of Calibre deck tarball? If I only want the results after P&R and do not care about LVS and DRC.

**Reply by harrisonliew** [reply, marked as answer]
Providing the tarball is [optional](https://github.com/ucb-bar/hammer/blob/5277152e35f664a23e816c26fcc66e605173b19a/hammer/technology/asap7/asap7.tech.json#L21). Just provide an invalid folder path for the `technology.asap7.tarball_dir` key.
